api/apps/api/src/modules/geo-features/geo-features.service.ts

```diff
@@ -854,6 +893,10 @@ export class GeoFeaturesService extends AppBaseService<
       .groupBy('fappu.feature_id')
       .getRawMany();
 
+    if (minAndMaxAmountsForFeatures.length === 0) {
+      return;
+    }
```

Comment:
Yeah I agree, it would be better to throw error, and catch it later. 
The only issue in this case is that by the time this function executes, where it gets executed, the changes relater to feature amounts have already been committed (geo-features.service.ts line 437 and features-amount-upload.service.ts line 116), and catching the error won't do much more than logging it (which already better than nothing, indeed)